pkg/cmd/repo/rename/rename.go

```diff
@@ -144,6 +148,21 @@ func renameRun(opts *RenameOptions) error {
 		fmt.Fprintf(opts.IO.Out, "%s Updated the %q remote\n", cs.SuccessIcon(), remote.Name)
 	}
 
+	if opts.DoConfirm && !opts.RenameLocalDir {
+		var confirmed bool
+		if confirmed, err = opts.Prompter.Confirm("Would you also like to rename the repo directory to ?", false); err != nil {
+			return err
+		}
+		if confirmed {
+			opts.RenameLocalDir = true
+		}
+	}
+	if opts.RenameLocalDir {
+		if err = renameRepoDir(newRepoName, opts); err != nil {
+			fmt.Fprintf(opts.IO.ErrOut, "Failed to rename local directory: %v", err)
+		}
+	}
```

Comment:
🤔 interesting idea putting this all in its own place. What's the UX look like for this? I was imagining something like:

```shell
# When the user doesn't specify anything
$ gh repo rename
? Rename <owner/repo> to: <new-name>
? Would you also like to rename the local directory to the same name? (y/N) y
✓ Renamed repository owner/<new-name> and local directory
✓ Updated the "origin" remote

# When the user specifies the repo name
$ gh repo rename <new-name>
? Rename <owner/repo> to <new-name>? (y/N) y
✓ Renamed repository owner/<new-name>
✓ Updated the "origin" remote

# When the user includes the --rename-dir flag
$ gh repo rename --rename-dir
? Rename <owner/repo> and local directory to: <new-name>
✓ Renamed repository owner/<new-name> and local directory
✓ Updated the "origin" remote

# When the user specifies the repo name and includes the --rename-dir flag
$ gh repo rename <new-name> --rename-dir
? Rename <owner/repo> to <new-name> and the local directory name? (y/N) y
✓ Renamed repository owner/<new-name> and local directory
✓ Updated the "origin" remote

# When the user specifies the repo name and includes the --rename-dir and --yes flags
$ gh repo rename <new-name> --rename-dir --yes
✓ Renamed repository owner/<new-name> and local directory
✓ Updated the "origin" remote
```

I'll add this UX to the issue 👍

Reply:
@jtmcg thanks for the clear UI, checkout the latest commit!